mesh/nurbs.cpp

```diff
@@ -1821,6 +1866,84 @@ void NURBSExtension::ConnectBoundaries()
    GenerateBdrElementDofTable();
 }
 
+void NURBSExtension::ConnectBoundaries1D(int bnd0, int bnd1)
+{
+   /*  int idx0 = -1, idx1 = -1;
```

Comment:
Remove commented code?

Reply:
I am working on 1D periodicity. I will post a small code changes later today.

Reply:
There are still some other issues with periodic BCs. That are being addressed in #2058